Tests/js/views/ez-userprofileview.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ User Profile view tests</title>
+</head>
+<body>
+<div class="container"></div>
+<div class="ez-view-usermenuview"><div class="ez-view-usermenuview"></div></div>
+
+<script type="text/x-handlebars-template" id="userprofileview-ez-template">
+    <div class="ez-user-profile">
+        <div class="ez-user-profile-info">
+            <p class="ez-user-profile-name">{{user.name}}</p>
+        </div>
+        <img class="ez-user-profile-image" src="{{#if user.avatar.uri}}{{user.avatar.uri}}{{else}}{{defaultAvatarImage}}{{/if}}" alt="{{avatarAltText}}">
```

Comment:
it's not aligned anymore with the actual template. On template, the good practice is to try to simplify it as much as possible in the test so it generates only what is needed for the view to work.